src/todos/todos.service.ts

```diff
@@ -1,4 +1,5 @@
 import { Injectable } from '@nestjs/common'
+import { QueryOptions } from 'mongoose'
```

Comment:
Business logic cannot depend on `mongoose`, it should be incapsulated in DB layer.